programs/server/config.xml

```diff
@@ -815,8 +1008,6 @@
     <!--
     <prometheus>
```

Comment:
Now we have 2 places to configure prometheus

Reply:
On the other hand, now we have a single place to configure listening interfaces.